### ISSUE TITLE
Passing GET parameters

### ISSUE THREAD

**Issue by FafMio** [Eredost/attributes-router#20]
Not possible to pass $_GET parameters ?

**Comment by booklet-mikolajczak** [comment]
Yep, parameters break match, I replaced in `match()` function line:

`$request = $_SERVER['REQUEST_URI'];`

to 

`$request = parse_url($_SERVER['REQUEST_URI'], PHP_URL_PATH); `

After remove query part, `matchRequest()` function work correct.
